src/Bit.BlazorUIPlayground.Web/Pages/About.razor.css

```diff
@@ -155,28 +154,29 @@
     box-sizing: border-box;
     color: #1A2027;
     transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
-    background-color: #FFFFFF;
+    background-color: var(--bit-clr-bg-primary);
```

Comment:
try not to use these css variables directly. instead, use the `BitCss` class to assign the required css class.